test/upgrade/upgrade_test.go

```diff
@@ -55,8 +55,8 @@ func TestServerlessUpgrade(t *testing.T) {
 					eventingupgrade.ContinualTest(),
 				},
 				kafkaupgrade.ChannelContinualTests(continual.ChannelTestOptions{}),
-				kafkaupgrade.SourceContinualTests(continual.SourceTestOptions{}),
```

Comment:
I can see that the SinkContinualTests also includes KafkaSource. Alright, not required to duplicate tests here.